site/src/Librecores/ProjectRepoBundle/DataFixtures/ORM/LoadDevTestData.php

```diff
@@ -118,4 +125,47 @@ private function createUser($username, $password)
 
         return $user;
     }
+
+    /**
+     * Populate an ClassificationHierarchy object
+     *
+     * @param integer $parent_id
+     * @param string $name
+     * @return \Librecores\ProjectRepoBundle\Entity\ClassificationHierarchy
+     */
+    private function createClassifier($parent_id,$name) {
+        $classifier = new ClassificationHierarchy();
+        $classifier->setParentId($parent_id);
+        $classifier->setName($name);
+        $classifier->prePersist();
+
+        return $classifier;
+    }
+
+   private $classifiers = array(
```

Comment:
- Class variables should go to the start of the file. 
- This could also be a constant. 
- Introduce a space after the `,`
- Add a comment what the first and second element in the array are for.